setup.py

```diff
@@ -25,14 +25,9 @@
     readme = fp.read()
 
 
-about = {}
-with open(os.path.join(here, 'substra', '__version__.py'), 'r', 'utf-8') as fp:
-    exec(fp.read(), about)
-
-
 setup(
     name='substra',
-    version=about['__version__'],
+    version='0.0',
     description='Substra CLI for interacting with substra-backend',
```

Comment:
This is not the same version, this refers to the release of the substra lib, not to the version of the backend API. We should keep the __version__.py file and its uses.